docs/modules/dataproxy/architecture.md

```diff
@@ -150,3 +150,51 @@ Sink configuration example and corresponding notes
     
     agent1.sinks.meta-sink-more1.max-survived-size = 3000000
     Maximum number of caches
+    
+# 4、Monitor metrics configuration instructions
+
+  DataProxy provide monitor indicator based on JMX, user can implement the code that read the metrics and report to user-defined monitor system.
+Source-module and Sink-module can add monitor metric class that is the subclass of org.apache.inlong.commons.config.metrics.MetricItemSet, and register it to MBeanServer. User-defined plugin can get module metric with JMX, and report metric data to different monitor system.
+
+  User can describe the configuration in the file "common.properties ". For example:
+  
+	metricDomains=DataProxy
+	metricDomains.DataProxy.domainListeners=org.apache.inlong.dataproxy.metrics.prometheus.PrometheusMetricListener
+	metricDomains.DataProxy.snapshotInterval=60000
```

Comment:
It's better to cover this in a code block or shell script block

Reply:
fixed， use shell script block.